### PR TITLE
Canonicool reminders

### PR DESCRIPTION
**Pull request by bethcollins92**
When merged, this PR will win us the Frankfurt hackathon :sunglasses:

### CODE REVIEW

File: scripts/canonicool-reminders.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+module.exports = async function (robot) {
+  robot.router.post("/hubot/canonicool-reminders", async function (req, res) {
+    // Rotate the presenters and send a message to the channel pinging the new ones.
+    const rotateRes = await axios.post(rotateURL, null, options);
+
+    const presentersData = await axios.get(presentersURL, options);
+    const presenters = presentersData.data;
+
+    const post_data = JSON.stringify({
```
Comment (albertkol) [inline scripts/canonicool-reminders.js:50]: :puke:

File: scripts/canonicool-reminders.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            JSON.stringify({
+              name: userName,
+            }),
+            options
+          );
+
+          presentersData = await axios.get(presentersURL, options);
+          presenters = presentersData.data;
+
+          const post_data = JSON.stringify({
```
Comment (albertkol) [inline scripts/canonicool-reminders.js:133]: :puke: :puke:

**Review by albertkol — COMMENTED** [review]
:+1:

File: scripts/canonicool-reminders.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        if (
+          reaction.emoji_name === "x" &&
+          reaction.user_id !== "q8yjh4wxupnw5jm5qw4omuw8zw"
+        ) {
+          cancelledUserIds.push(reaction.user_id);
+        }
+      });
+
+      const cancelledUserNames = [];
+      var userRes;
```
Comment (mrgnr) [inline scripts/canonicool-reminders.js:107]: Use `let` instead of `var`.

**Review by ClementChaumel — APPROVED** [review]

**Pull request merged**